internal/appsec/config.go

```diff
@@ -109,6 +121,25 @@ func readRateLimitConfig() (rate uint) {
 	return uint(parsed)
 }
 
+func readObfuscatorConfig() *ObfuscatorConfig {
+	keyRegex, kPresent := os.LookupEnv(obfuscatorKeyEnvVar)
+	valueRegex, vPresent := os.LookupEnv(obfuscatorValueEnvVar)
+
+	if keyRegex == "" && !kPresent {
+		log.Info("appsec: starting with the default regexp for matched parameter keys obfuscation")
+		keyRegex = defaultObfuscatorKeyRegex
+	} else {
+		log.Info("appsec: starting with user regexp for matched parameter keys obfuscation")
```

Comment:
Since we have RE2 in Go too, let's validate that the regexp compiles so we can log an error here, without relying on the WAF logs for now.